MAINTAINERS.md

```diff
@@ -0,0 +1,122 @@
+# Maintainer Terms & Guidelines
+
+Kiwi is an open source activity- & participation platform, intended for use by
+groups in all shapes and sizes. As such, it's continously developed by these 
+different groups of people. As any open source project alike, open source is
+about collaboration. However, different users may have different needs and
+different priorities. To bring these differing views together, the following
+terms and guidelines were devised to bring everyone together on equal grounds.
+
+This project is intended for use by organised groups of people: associations
+organisations, sports clubs or anything alike. Regardless of group size, each
+of these entities has a unique perspective that should be weighed equally to 
+any other. Therefore, a group of people (hereafter referred to as organisation) 
+may apply for maintainer status. With maintainer status, your organisation will
+be able to weigh in on the direction and priorities of the project. In return, 
+and in the spirit of free, open source software, we ask that your organisation 
+actively participates in the development and maintenance of the project. (Note: 
+individuals that are not affiliated to any organisation are welcome to apply 
+for maintainer status too.)
+
+## Terms of maintainer status
+Each entity with maintainer status shall provide one or more maintainers, who'll
+participate in the development. They're always the point of contact, except for
+notices on the maintainer status of the organisation or for notices on changes
+in the list the maintainers representing an organisation. All communication
+among maintainers takes place through an online channel, indicated after being
+granted maintainer status.
+
+At least once every six months, a vote is held on the priorities and agreements
+for the next six months. A vote will be announced by the project owner at least
+two weeks in advance, through the online communication channel for maintainers,
+distinctly from any other communication. Any maintainer may introduce topics, up
+to week before the vote.
+
+Each entity with maintainer status gets a single vote. If multiple maintainers
+representing a single entity are present, only of them may cast a vote on behalf
+of the entity they represent. For any changes to these terms and guidelines, as
+well as the contribution guidelines and the code of conduct, a vote must be held.
+
+Maintainers are expected to be active, supportive and responsive. Upon questions
+or requests from fellow maintainers, a response or indication of acknowledgement
+of the request must be given within 72 hours. If a maintainer is unavailable for
+a longer period of time, a notice to must be provided accordingly. Maintainers
+are expected to adhere to all deadlines and agreements. General development
+meetings (whose schedule is decided upon during votes) shall be regularly
```

Comment:
could be that I am wrong here, is "whose schedule" the correct phrasing?

Reply:
[Arguably](https://en.wikipedia.org/wiki/Inanimate_whose)